go.mod

```diff
@@ -0,0 +1,46 @@
+module github.com/cyberark/terraform-provider-conjur
+
+require (
+	github.com/apparentlymart/go-dump v0.0.0-20190214190832-042adf3cf4a0 // indirect
```

Comment:
Lots of indirect refs which I guess we picked up from the conversion. While this is more stable, in the future we should just define the toplevel refs and clean this up if we can but deff not a blocker for the changeover.